packages/lisk-transactions/test/1_register_second_passphrase.ts

```diff
@@ -132,7 +135,7 @@ describe('#registerSecondPassphrase transaction', () => {
 			).and.not.be.empty;
 		});
 
-		it('should not have a signSignature property', () => {
+		it.skip('should not have a signSignature property', () => {
```

Comment:
why was it skipped?
maybe checked to be `undefined`?

Reply:
Was skipped to get this kind of feedback. 💯
Will check for undefined